pyfftw/pyfftw.pyx

```diff
@@ -1408,6 +1436,11 @@ cdef class FFTW:
         copy the returned array.
         '''
 
+        if ortho is None:
+            ortho = self._ortho
+        if normalise_idft is None:
+            normalise_idft = self._normalise_idft
+
         if ortho and normalise_idft:
             raise ValueError('Invalid options: ortho and normalise_idft cannot'
                              ' both be True.')
```

Comment:
I am still not sure I see the problem.  After 1439-1442 `ortho` and `normalise_idft` will both be either `True` or `False` as they will now have the values from the `__cinit__` constructor which requires bools (I just updated the `__cinit__` parameters to have type `bint` which I had previously neglected to do).

Please suggest a specific fix if I am still missing the point here